requirements.txt

```diff
@@ -17,4 +17,6 @@ zipp==3.11.0
 pyOpenSSL==22.1.0
 typing-extensions==4.5.0 
 parameterized==0.9.0
+# Future PR - use poetry in MongoDB Charm
+poetry==1.8.2
```

Comment:
this is a temp fix for the unit tests - in the future we should have a separate PR to bring the whole MongoDB repo to use poetry